ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1746,6 +1746,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-stack-ipi
+- as: azure-stack-ipi-proxy-p3-f28
```

Comment:
How about add one more job for 4.12

Reply:
okay, will change to p2.
Bug [2104997](https://bugzilla.redhat.com/show_bug.cgi?id=2104997) is only fixed on 4.13 now, and not backport yet. Will add 4.12 job when bug get fixed.

Reply:
ah. got it